README.md

```diff
@@ -12,7 +12,7 @@ And invoking as:
 fbMotion1(stMotionStage:=M1);
 ```
 
-You would then link your hardware as appropriate to `M1.Axis`, `M1.bLimitForwardEnable`, `M1.bLimitBackwardEnable`, `M1.bHome`, `M1.bBrakeRelease`, and `M1.bHardwareEnable`.
+You would then link your hardware as appropriate to `M1.Axis`, `M1.bLimitForwardEnable`, `M1.bLimitBackwardEnable`, `M1.bHome`, `M1.bBrakeRelease`, and `M1.bHardwareEnable`. It is important to set `M1.bEnableMode` the value the most fits your use case (see settings below). You also need to set `bPowerSelf` to `TRUE`, unless your device participates in LCLS's PMPS system, where it must be `FALSE`.
```

Comment:
Minor typo: should be `M1.nEnableMode`, correct?

Reply:
Correct, thanks for spotting this. I've fixed the typo.